lib/my_john_deere_api/errors/missing_location_header_error.rb

```diff
@@ -0,0 +1,21 @@
+module MyJohnDeereApi
+  ##
+  # This error is used when an HTTP response does not contain
+  # an expected Location header.
+
+  class MissingLocationHeaderError < StandardError
```

Comment:
Instead of this we can probably create the `ConflictError`

Reply:
I prefer to create/return the more specific error because it will tell the developer exactly what went wrong, whereas "ConflictError" is a little more ambiguous. Does that sound good, or is there something (like a common convention) that I'm missing?

Reply:
I can definitely agree that `ConflictError` is ambiguous, but so is the error it's hoping to represent. Is the 409 error we encountered and this error class representative of the same thing?

Reply:
These could be complete overlap. I believe the only reason we ever got a response without a location header when we expected one was because we were getting 409 errors. But I'm hesitant to get rid of it, since it does cover the base of a missing header. I could technically remove it and I think we'd be fine. How would you like to handle this?

Reply:
I believe we can probably skip this PR altogether. So If I can get approval or final thoughts on PR #5, we could push the gem today.